pages/initiatives/gsoc/gsoc2022ideas.md

```diff
@@ -5,7 +5,7 @@ tags: gsoc
 permalink: /initiatives/gsoc/gsoc2022ideas
 ---
 
-# {{page.title}}
```

Comment:
This was correct it’s using a front matter variable

Reply:
Oh, I came to this page from google search results and found this odd. I checked the [gsoc2021ideas.md](https://github.com/OWASP/www-community/blob/master/pages/initiatives/gsoc/gsoc2021ideas.md) and submitted the PR according to that. Shall I close this now?

Reply:
No don't close it. Let's just fix it properly ;)